inc/class-main.php

```diff
@@ -83,6 +83,8 @@ public function autoload_classes() {
 			'\ThemeIsle\GutenbergBlocks\Integration\Form_Email',
 			'\ThemeIsle\GutenbergBlocks\Server\Form_Server',
 			'\ThemeIsle\GutenbergBlocks\Server\Prompt_Server',
+			'\ThemeIsle\GutenbergBlocks\Template_Cloud',
```

Comment:
We usually put namespace new features like this as part of [Plugins](https://github.com/Codeinwp/otter-blocks/tree/master/inc/plugins).